score-lang-compiler/src/main/java/org/openscore/lang/compiler/parser/YamlParser.java

```diff
@@ -42,5 +46,19 @@ public ParsedSlang parse(SlangSource source) {
             throw new RuntimeException("There was a problem parsing the YAML source: " + source.getName() + ".\n" + e.getMessage(), e);
         }
     }
+    public Map<String, Serializable> parseInputFile(SlangSource source){
+        Validate.notEmpty(source.getSource(), "Source " + source.getName() + " cannot be empty");
+
+        try {
+            @SuppressWarnings("unchecked") Map<String,Serializable> inputs =  (Map<String, Serializable>)yaml.load(source.getSource());
+            if(inputs == null) {
+                throw new RuntimeException("Source " + source.getName() + " does not contain YAML content");
+            }
+            return inputs;
+        } catch (Throwable e) {
+            throw new RuntimeException("There was a problem parsing the YAML source: " + source.getName() + ".\n" + e.getMessage(), e);
+        }
+    }
+
```

Comment:
Should this method be added someplace else? Since it doesn't actually use ParsedSlang or parses anything. It just loads the yaml file.